--- ProjectCenter/PCLib/PCFileManager.m.orig	2003-07-05 13:58:52.000000000 +0200
+++ ProjectCenter/PCLib/PCFileManager.m	2003-10-12 13:32:26.506892424 +0200
@@ -33,7 +33,7 @@
 #include "PCFileManager+UInterface.h"
 
 #if defined(GNUSTEP)
-#include <AppKit/IMLoading.h>
+#include <GNUstepGUI/IMLoading.h>
 #endif
 
 @implementation PCFileManager
--- ProjectCenter/PCLib/PCProjectManager.m.orig	2003-07-02 13:32:45.000000000 +0200
+++ ProjectCenter/PCLib/PCProjectManager.m	2003-10-12 13:32:58.632008664 +0200
@@ -38,7 +38,7 @@
 #include "PCProjectManager+UInterface.h"
 
 #if defined(GNUSTEP)
-#include <AppKit/IMLoading.h>
+#include <GNUstepGUI/IMLoading.h>
 #endif
 
 #define SavePeriodDCN @"SavePeriodDidChangeNotification"
