--- ProjectCenter-0.4.3/GNUmakefile.preamble.orig	2005-11-18 17:27:44.000000000 +0100
+++ ProjectCenter-0.4.3/GNUmakefile.preamble	2006-02-05 17:15:06.831749000 +0100
@@ -49,7 +49,7 @@
 ADDITIONAL_LDFLAGS +=
 
 # Additional library directories the linker should search
-ADDITIONAL_LIB_DIRS += -L./Library/ProjectCenter.framework/Versions/Current \
+ADDITIONAL_LIB_DIRS += -L./Library/ProjectCenter.framework/Versions/Current/$(ARCH_OBJ_DIR)/$(LIBRARY_COMBO) \
                        -L./Library/ProjectCenter.framework
 
 # Additional LDFLAGS to pass to the linker
